static/js/components/ProfileImageUploader.js

```diff
@@ -76,7 +76,7 @@ const ProfileImageUploader = ({
         type='button'
         className={`save-button ${photo ? 'primary-button' : 'secondary-button disabled'}`}
         key="save"
-        onClick={updateUserPhoto}>
+        onClick={photo ? updateUserPhoto : undefined}>
```

Comment:
I guess not actually, in HTML if the `onclick` is missing it's treated as if the function weren't there